scapy/route.py

```diff
@@ -126,54 +125,55 @@ def ifadd(self, iff, addr):
         the_net = the_rawaddr & the_msk
         self.routes.append((the_net, the_msk, '0.0.0.0', iff, the_addr, 1))
 
-    def route(self, dest, verbose=None):
-        if dest is None:
-            dest = "0.0.0.0"
-        elif isinstance(dest, bytes):
+    def route(self, dst, verbose=conf.verb):
```

Comment:
I think it is helpful that `route(None)` gives the default route.

Reply:
Could be, but that’s inconsistent with route6 :/ should I add it for both ?

Reply:
That's my opinion yes